website/www/site/data/en/graphic.yaml

```diff
@@ -10,13 +10,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- title: Sources
+- title: Data Sourcing
   body: Beam reads your data from a diverse set of supported sources, no matter if it’s on-prem or in the cloud.
   icon: /images/model-sources-icon.svg
-- title: Processing
-  body: Beam executes your business logic for Batch and Streaming use cases.
```

Comment:
There's no need to capitalize "Batch" and "Streaming".

Reply:
changed